sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/models/BaseOptions.java

```diff
@@ -0,0 +1,39 @@
+package com.microsoft.appcenter.storage.models;
+
+import java.util.Calendar;
+
+public abstract class BaseOptions {
+
+    public static final int INFINITE = -1;
+    public static final int NO_CACHE = 0;
+    public static final int DEFAULT_ONE_HOUR = 60 * 60;
+
+    protected BaseOptions() {
+        this.mTtl = DEFAULT_ONE_HOUR;
+    }
+
+    protected BaseOptions(int ttl) {
+        if (ttl < -1) throw new IllegalArgumentException("Time-to-live should be greater or equal to zero or -1 for infinite.");
+        this.mTtl = ttl;
+    }
+
+    private int mTtl;
+
+    /**
+     * @return document time-to-live in seconds (default to 1 hour)
+     */
+    public int getDeviceTimeToLive() { return mTtl; }
```

Comment:
Format the file, please always always commit via Android Studio UI:

<img width="268" alt="screen shot 2019-03-04 at 5 18 14 pm" src="https://user-images.githubusercontent.com/623672/53773996-8398d980-3ea1-11e9-8fa1-cc1f3407acae.png">